passmark/base_test_results/test1/verify

```diff
@@ -0,0 +1,27 @@
+NumTestProcesses:[ 0-9. ]{1,}$
+CPU_INTEGER_MATH:[ 0-9. ]{1,}$
+CPU_FLOATINGPOINT_MATH:[ 0-9. ]{1,}$
+CPU_PRIME:[ 0-9. ]{1,}$
+CPU_SORTING:[ 0-9. ]{1,}$
+CPU_ENCRYPTION:[ 0-9. ]{1,}$
+CPU_COMPRESSION:[ 0-9. ]{1,}$
+CPU_SINGLETHREAD:[ 0-9. ]{1,}$
+CPU_PHYSICS:[ 0-9. ]{1,}$
+CPU_MATRIX_MULT_SSE:[ 0-9. ]{1,}$
+CPU_mm:[ 0-9. ]{1,}$
+CPU_sse:[ 0-9. ]{1,}$
+CPU_fma:[ 0-9. ]{1,}$
+CPU_avx:[ 0-9. ]{1,}$
+CPU_avx512:[ 0-9. ]{1,}$
```

Comment:
avx512 not present on ARM.  This would fail.

Consider alternate ways to verify without being test specific